src/Microsoft.SqlTools.ServiceLayer/Migration/MigrationService.cs

```diff
@@ -435,15 +482,17 @@ internal async Task<MigrationAssessmentResult> GetAssessmentItems(string[] conne
             SqlAssessmentConfiguration.ReportsAndLogsRootFolderPath = Path.GetDirectoryName(Logger.LogFileFullPath);
             DmaEngine engine = new DmaEngine(connectionStrings);
             ISqlMigrationAssessmentModel contextualizedAssessmentResult = await engine.GetTargetAssessmentResultsListWithCheck(System.Threading.CancellationToken.None);
-            engine.SaveAssessmentResultsToJson(contextualizedAssessmentResult, false);
+            var assessmentFileName = String.Format("SqlAssessmentReport-{0}.json", DateTime.UtcNow.ToString("yyyyMMddHHmmss"));
```

Comment:
the file name should be provided by customer? if not, actually the NuGet will provide the default name like SqlAssessmentReport-{0}.json

Reply:
yeah filename is very generic. ex - SqlAssessmentReport-20220602
should we add instance name etc? let's discuss offline